### DISCUSSION TITLE
How to Add /docs to URL Without Creating a Nested Folder in Starlight Docs?

### DISCUSSION THREAD

**Discussion by temiye18** [withastro/starlight#2447] [Q&A] [answered]
Hi all!  

I have an Astro website that has starlight docs as a subdomain, I am trying to have have a `/docs` in my url without having to create another `docs` folder inside starlight's default `docs` folder as seen in the screenshot below... please how can accomplish this?

![ksnip_20241008-132400](https://github.com/user-attachments/assets/83d9854a-71f0-4092-a3be-b7a368e25a42)

**Comment by delucis** [comment, marked as answer]
That is not currently supported. A nested `docs/` directory is the recommended approach at the moment as documented in https://starlight.astro.build/manual-setup/#use-starlight-at-a-subpath 

It is possible to manually set a `slug` in a page’s frontmatter, e.g.:

```md
---
title: My page
slug: docs/page
---
```

But that’s obviously not really a great approach at scale.

There’s some more discussion of ideas to improve this here: https://github.com/withastro/starlight/discussions/966

**Comment by temiye18** [comment]
Thanks for your response. I really appreciate it
